portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Rebeka Dittrich</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="socialnav">
+      <a href="https://uk.linkedin.com" target="_blank">LinkedIn<a>
+      <a href="https://github.com/rebekadittrich" target="_blank">GitHub</a>
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="mainnav">
+          <a class="outline" href="index.html">About me</a>
+          <a href="projects.html">Resume</a>
+          <a href="gallery.html">Gallery</a>
+        </div>
+      </div>
+      <div class="column">
+        <div id="content">
+          <h1>Hello.<br> I am Rebeka</h1>
+          <img src="/images/portfolio1.jpg" alt="Cat" style="width:500px;height:500px;">
+          <div class="text-content">
+            <p class="text-content">I'm a second year student at Imperial College London from Budapest, Hungary. 
```

Comment:
> I'm a second year student at Imperial College London from Budapest, Hungary

The sentence seems grammattically ambiguous, I suggest to split it in two (optional).